index.html

```diff
@@ -54,6 +55,174 @@ <h2 class="connect">LET'S CONNECT</h2>
+				<h2 class="tonic3">Tonic</h2>
+				<div class="grid-container3">
+				  <div class="grid-13">
+					<p>CANOPY</p>
+				  </div>
+				  <div class="grid-23">
+					<p>.</p>
+				  </div>
+				  <div class="grid-33">
+					<p>Back End Dev</p>
+				  </div>
+				  <div class="grid-43">
+					<p>.</p>
+				  </div>
+				  <div class="grid-53">
+					<p>2015</p>
+				  </div>
+				</div>
+
+				<p class="day3">
+					A daily selection of privately personalized reads; 
+					no accounts or sign-ups required.
+				</p>
+				<ul class="langs3">
+					<li class="btn13">html</li>
+					<li class="btn23">css</li>
+					<li class="btn33">javaScript</li>
+				</ul>
+				<button class="btn43" type="submit">See Project</button>
+			</div>
+			<div class="works4">
+				<img class="snap4" src="images/snapshot4.svg" alt="">
+				<h2 class="tonic4">Multi-Post Stories</h2>
+				<div class="grid-container4">
+				  <div class="grid-14"><p>CANOPY</p></div>
+				  <div class="grid-24"><p>.</p></div>
+				  <div class="grid-34"><p>Back End Dev</p></div>
+				  <div class="grid-44"><p>.</p></div>
+				  <div class="grid-54"><p>2015</p></div>
+				</div>
+				<p class="day4">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+				<ul class="langs4">
+					<li class="btn14">html</li>
+					<li class="btn24">css</li>
+					<li class="btn34">javaScript</li>
+				</ul>
+				<button class="btn44" type="submit">See Project</button>
+			</div>
+		</section>
+	<div class="div-class">
+		<section class="about-myself">
+      		<div class="desk-about">
+        		<h1 class="aboutmy">About <br> Myself</h1>
+        		<p class="am">I’m a software developer! I can help you build a product , feature or website Look through some of my
+	          	work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+	          	contact me.</p>
+        		<h3 class="connect">LET'S CONNECT</h3>
+        		<ul class="icons">
+                    <li class="twitter"><img src="images/twitter.svg" alt="twitter svg"></li>
+					<li class="vector1"><img src="/images/vector.svg" alt="vector"></li>
+					<li class="vector2"><img src="images/Vector.svg" alt="vector svg"></li> 
+					<li class="medium"><img src="images/medium.svg" alt="medium svg"></li>
+					<li class="github"><img src="images/github.svg" alt="github svg"></li>
+					<li class="angelist"><img src="images/angellist.svg" alt="angellist"></li>
+        		</ul>
+        		<button class="resume">Get My Resume</button>
+      		</div>
+		    <div class="expertise">
+		        <ul class="language">
+		          <li>
+		            <p>Languages</p>
+		          </li>
+		          <li><i class="fa fa-chevron-down" aria-hidden="true"></i></li>
+		        </ul>
+		        <div class="desk-programming">
+		          <ul class="outer-item">
+		            <li class="programming"><img src="images/ELLIPSE.svg" alt="JavaScript"></li>
```

Comment:
Kindly fix the logo for Javascript 😃

Your Design:
![image](https://user-images.githubusercontent.com/73714615/185757432-13be2861-5b91-4ec6-9ca6-53876afca8c9.png)

Figma Design:
![image](https://user-images.githubusercontent.com/73714615/185757455-36fa66c0-37b8-43fd-a16c-a6c77b510a09.png)